shared/fs/common/path-item-action/header.js

```diff
@@ -34,9 +34,9 @@ class Header extends React.PureComponent<Props> {
   render() {
     return (
       <Kb.Box2 direction="vertical" fullWidth={true} centerChildren={true} style={styles.container}>
-        <PathItemIcon path={this.props.path} size={32} style={styles.pathItemIcon} />
-        <StaticBreadcrumb pathElements={Types.getPathElements(this.props.path)} />
+        <PathItemIcon path={this.props.path} size={48} style={styles.pathItemIcon} />
         <Kb.Box2 direction="horizontal" style={styles.nameTextBox}>
+          {Styles.isMobile && <StaticBreadcrumb pathElements={Types.getPathElements(this.props.path)} />}
```

Comment:
Did you mean to move this inside this box?

Reply:
No.. fixing.